lib/interfaces.ts

```diff
@@ -15,6 +15,7 @@ interface RedisStreamOptionsXreadGroup {
   block?: number;
   consumerGroup: string;
   consumer: string;
+  deleteAfterAck?: boolean;
```

Comment:
I think `deleteMessagesAfterAck` is a better name.

Reply:
Ok for me. I copy from Symfony Messenger.